Backend/src/Model.ts

```diff
@@ -0,0 +1,49 @@
+export interface Task {
```

Comment:
The next structure will be probably appropriate for the future:
- src
-- interfaces
-- services

https://stackoverflow.com/questions/36633033/how-to-organize-typescript-interfaces

Every interface/service it's better to keep in the separate file.